app/views/step_by_step_pages/show.html.erb

```diff
@@ -17,20 +17,15 @@
 
 <%= render 'shared/steps/form_notice', message: notice, status: "success" %>
 
-<h1><%= @step_by_step_page.title %></h1>
+<%= render 'shared/steps/heading',
+  step_nav: @step_by_step_page.title,
+  action: 'Edit steps'
+%>
 
-<ul class="list-inline step-actions">
-  <li><%= link_to 'Add a new step', new_step_by_step_page_step_path(@step_by_step_page), class: "btn btn-primary" %></li>
-  <li><%= link_to 'Preview', preview_url(@step_by_step_page.slug, auth_bypass_id: @step_by_step_page.auth_bypass_id), class: "btn btn-primary" %></li>
-  <li><%= link_to 'Edit title, intro or slug', edit_step_by_step_page_path(@step_by_step_page), class: "btn btn-primary" %></li>
-  <li><%= link_to 'Choose navigation', step_by_step_page_navigation_rules_path(@step_by_step_page), class: "btn btn-primary" %></li>
-  <% if @step_by_step_page.steps.length > 0 %>
-    <li><%= link_to "Reorder steps", step_by_step_page_reorder_path(@step_by_step_page), class: "btn btn-primary" %></li>
-  <% end %>
-</ul>
+<%= render 'nav', step_by_step_page: @step_by_step_page, active:"edit-steps" %>
 
 <div class="row">
-  <div class="col-md-7">
+  <div class="col-md-12">
```

Comment:
How is this change related to the change to tabbed navigation?

Reply:
The tabbed nav is full width, the content, therefore, looks weird and out of place when it is not full width

Reply:
Can you add this explanation to the commit message?